hasher-matcher-actioner/hmalib/models.py

```diff
@@ -38,6 +39,33 @@ def remove_content_key_prefix(key: str) -> str:
         return key[len(DynamoDBItem.CONTENT_KEY_PREFIX) :]
 
 
+class SNSMessage:
+    def to_sns_message(self) -> str:
+        raise NotImplementedError
+
+    @classmethod
+    def from_sns_message(cls, message: str) -> "SNSMessage":
+        raise NotImplementedError
+
+
+@dataclass
+class Label:
```

Comment:
TODO make name more descriptive? What's this used for?

Reply:
@jeberl Label are a first class entity and I expect them to be used throughout the 'pipelines' to attach more information to a match. Attaching labels decouples subsequent components from any data models specific to preceeding components. Any qualifier more descriptive here would mean we limit what Labels are used for.

I am strongly for retaining the vanilla 'Label' name. Let's discuss if you have concerns with this.